docs/about-stacks.md

```diff
@@ -24,47 +24,44 @@ A terraform stack is a runnable terraform module that operates on a subset of
 the infrastructure's resources.
 ```
 
-By _runnable_ it means it has everything needed to call
-`terraform init/plan/apply` . For example, a module used by other modules, ie,
-don't have the `provider` explicitly set, is not runnable hence it's
-**not a stack**.
+For a module to be considered a stack, it must be complete and self-contained, 
+meaning that it can run all the necessary Terraform commands (`init/plan/apply`) 
+without any external dependencies.
 
-If your runnable Terraform module creates your whole infrastructure, *it's
-also not a stack*, since the idea of stacks is to be a unit of infrastructure
-decomposition.
+If a module is missing any required components, such as provider settings, 
+it cannot be considered a stack.
 
-## Isolate code with higher change frequency
+It's also worth noting that if a module creates your entire infrastructure, 
+it's not a stack. **Stacks are designed to break down your infrastructure into** 
+**manageable pieces**, so they operate on a specific subset of resources rather 
+than your entire infrastructure.
 
-If you have a high change frequency in your infrastructure, for example, several
-deploys per day/week or several configuration changes per day/week, then if you
-apply the change in your single Terraform project, some dependent resources can
-be recreated leading to increased downtime. There are several reasons why a
-dependent module can be recreated, eg.: attributes with variable interpolation
-from recreated resources; underspecified attributes refreshing during plan, etc.
+## Isolate code that changes frequently
 
-By using stacks you can modify only the stacks affected by the deploys or
-configuration changes needed, but you have to choose the size of the stack
-wisely to avoid duplication.
+If you make a lot of updates to your system, like deploying new code or changing 
```

Comment:
*your system* is way off here, no? 
*System* in computing is often used when referring to software that's part of the operating system.